dvc.yaml

```diff
@@ -0,0 +1,116 @@
+stages:
+  tiler:
+    cmd: C:\Develop\Python\Python37\python.exe ..\..\tools\Tiler\tiler.py
```

Comment:
If you add python to PATH this could be `cmd: python ..\..\tools\Tiler\tiler.py`

Reply:
For some reason, even though I added python37 to my `PATH` this stage fails when I try to run it with `python` instead of the whole path. But I try to note it on the pipeline readme

Reply:
Update : 
I added a vars list in dvc.yml where the local python path is included so it can be used as a var in the stages